Signal/src/view controllers/OWSLinkedDevicesTableViewController.m

```diff
@@ -248,6 +249,15 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
     }
 }
 
+-(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
+{
```

Comment:
I didn't do that change. If you insist you want it that way I will update it but in my opinion with that window it is more elegant to keep it as it is because if I move permission check to the destination controller, the actual permission check will take place with new screen already opened and if user denies permissions it will stay with screen containing empty placeholder for camera. The other screen that looks similar has camera part pulled down (at least I remembered it that way) and this one has it opened during loading.